.github/ISSUE_TEMPLATE/config.yml

```
@@ -0,0 +1,19 @@
+blank_issues_enabled: false
+
+contact_links:
+  - name: Documentation
+    url: https://github.com/qgis/QGIS-Documentation/issues
+    about: Visit the GitHub documentation repository.
```

```suggestion
    about: >
       If the issue concerns QGIS manuals then you'd need to report in the GitHub QGIS-Documentation repository.
```
I'd like to avoid people reporting API docs (c++ and pyqgis websites) issues in the manuals repo and wanted  to use https://docs.qgis.org/ (which is the root url of what the docs repo manages) but it does not directly bring to docs, so not sure it'd have been clearer.